util/generate-ami-list.py

```diff
@@ -65,7 +65,11 @@ def get_ami_list_from_ec2(main_region, regions, date, cookbook_git_ref, node_git
         elif cookbook_git_ref and node_git_ref:
             filters.append({"Name": "tag:parallelcluster_cookbook_ref", "Values": ["%s" % cookbook_git_ref]})
             filters.append({"Name": "tag:parallelcluster_node_ref", "Values": ["%s" % node_git_ref]})
-            filters.append({"Name": "name", "Values": ["aws-parallelcluster-*"]})
+            if build_date:
```

Comment:
minor: you can write:
```python
filters.append({"Name": "name", "Values": ["aws-parallelcluster-*%s" % build_date if build_date else ""]})
```

Reply:
Thanks for the suggestion.
Btw the correct line is
```
filters.append({"Name": "name", "Values": ["aws-parallelcluster-*%s" % (build_date if build_date else "")]})
```
otherwise you end up in having a Values="" if build_date is not defined